desc/magnetic_fields/_core.py

```diff
@@ -2658,6 +2671,58 @@ def odefun(s, rpz, args):
             [r * br / bp * jnp.sign(bp), jnp.sign(bp), r * bz / bp * jnp.sign(bp)]
         ).squeeze()
 
+    @jit
+    def odefun_iota(s, rpzt, args):
```

Comment:
per discussion last week this should probably be moved to a global scope to avoid recompiling each time. Same for the original `odefun`